packages/flutter/lib/src/material/text_selection_toolbar.dart

```diff
@@ -117,66 +131,6 @@ class TextSelectionToolbar extends StatelessWidget {
   }
 }
 
-// Positions the toolbar at the given anchor, ensuring that it remains on
-// screen.
-class _TextSelectionToolbarLayoutDelegate extends SingleChildLayoutDelegate {
```

Comment:
This has been moved to its own file and made public to be shared here and with CupertinoTextSelectionToolbar. See packages/flutter/lib/src/widgets/text_selection_toolbar_layout_delegate.dart.